### ISSUE TITLE
Not showing battery percentage of the connected Bluetooth device 

### ISSUE THREAD

**Issue by AbhishekAglave** [ubuntu/ubuntu-report#52]
Show the battery percentage of the connected Bluetooth device if applicable for the device type if it runs on the battery.
This improvement must be made because most people use wireless headphones or earphones and to check if the battery is sufficient we have to disconnect the device and connect it to the mobile phone.

**Comment by AbhishekAglave** [comment]
Please tell if it is possible to check.

**Comment by Darshan-Padia** [comment]
Yes, you can see the battery status of connected Bluetooth devices in Ubuntu 23.10 and later by going to the Settings > Power pane. You can also see the battery percentage in the top bar by following these steps:
Open the Activities overview
Type Power
Click Power to open the panel
Switch the Show Battery Percentage switch to on

**Comment by sovetski** [comment]
I see the battery but not the percentage

![Image](https://github.com/user-attachments/assets/673a577b-7320-4645-bfb6-5e84e6743712)
